Dark theme renders dark text in input fields

<!-- NOTE: Questions will be converted to Discussions. You can find them at https://github.com/nextauthjs/next-auth/discussions! -->

**Your question**
I suspect this is a bug, but wanted to follow protocol and submit it as a question first: 

When adding next-auth 3.2.0, `options.theme` defaults to `auto`. When system theme is set to dark, button text will be light but input field text will be dark rendering it invisible. Below is an example with the email provider enabled:

![image](https://user-images.githubusercontent.com/147237/106940633-c992f380-66ef-11eb-80c3-2542a998a40c.png)


**What are you trying to do**
<!-- A description of what you are trying to do, for context. -->

**Reproduction**
<!-- If your question is code related, adding a reproduction to your use case can greatly reduce the time it takes us to figure Bare bones install with `Providers.Email` added. 

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [ ] Could not find relevant documentation
* [ ] Found the example project helpful
* [ ] Did not find the example project helpful

I'd be glad to! Though I'm unable to get `npm run dev` running locally as its failing on missing babel and postcss - are these meant to be installed globally? 

```❯ npm run dev

> next-auth@0.0.0-semantically-released dev
> next | npm run watch:css

sh: next: command not found

> next-auth@0.0.0-semantically-released watch:css /Users/mikecavaliere/Sandbox/next-auth
> postcss --config config/postcss.config.js --watch src/**/*.css --base src --dir dist

sh: postcss: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! next-auth@0.0.0-semantically-released watch:css: `postcss --config config/postcss.config.js --watch src/**/*.css --base src --dir dist`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the next-auth@0.0.0-semantically-released watch:css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mikecavaliere/.npm/_logs/2021-02-05T01_20_31_792Z-debug.log
npm ERR! code 1
npm ERR! path /Users/mikecavaliere/Sandbox/next-auth
npm ERR! command failed
npm ERR! command sh -c next | npm run watch:css

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mikecavaliere/.npm/_logs/2021-02-05T01_20_31_836Z-debug.log

╭─ ~/Sandbox/next-auth  canary !1 ─────────────────────────────────────────────────── 1 ✘  2.6.5 
╰─















```

--- a/src/css/index.css
+++ b/src/css/index.css
@@ -80,6 +80,7 @@ input[type] {
   font-size: 1rem;
   border-radius: var(--border-radius);
   box-shadow: inset 0 .1rem .2rem rgba(0, 0, 0, .2);
+  color: var(--color-text);
 
   &:focus {
     box-shadow: none;

